scissors-sample/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,9 @@
         android:id="@+id/crop_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:cropviewViewportHeightRatio="1"
+        app:cropviewRatio="0.8"
```

Comment:
How does ratio compare to heightRatio? Are they two separate things? Is this widthRatio?